static/src/stylesheets/layout/new-header/_header-cta-item.scss

```diff
@@ -28,6 +28,13 @@
     @include mq(tablet) {
         font-size: 17px;
     }
+
+    .new-header--slim & {
```

Comment:
I like the way you did it - wouldn't change it for now, because all the rest is structured that way.